build.sbt

```diff
@@ -1,96 +1,36 @@
-import sbtcrossproject.CrossPlugin.autoImport.crossProject
-
-inThisBuild(Seq(
-  name := "Scala DOM Types",
-  normalizedName := "domtypes",
-  organization := "com.raquo",
-  scalaVersion := "2.12.12",
-  crossScalaVersions := Seq("2.11.12", "2.12.12", "2.13.4")
-))
-
-// @TODO[WTF] Why can't this be inside releaseSettings?
-releaseCrossBuild := true
-
-// @TODO[SBT] How to extract these shared settings into a separate release.sbt file?
-lazy val releaseSettings = Seq(
-  homepage := Some(url("https://github.com/raquo/scala-dom-types")),
-  licenses += ("MIT", url("https://github.com/raquo/scala-dom-types/blob/master/LICENSE.md")),
-  scmInfo := Some(
-    ScmInfo(
-      url("https://github.com/raquo/scala-dom-types"),
-      "scm:git@github.com/raquo/scala-dom-types.git"
+ThisBuild / scalaVersion := ScalaVersions.v213
+ThisBuild / crossScalaVersions := Seq(ScalaVersions.v3RC1, ScalaVersions.v213, ScalaVersions.v212, ScalaVersions.v211)
+
+lazy val domtypes =
+  crossProject(JSPlatform, JVMPlatform).in(file("."))
+    .settings(
+      scalacOptions ~= (_.filterNot(Set(
+        "-Wunused:params",
+        "-Ywarn-unused:params",
+        "-Wunused:explicits"
+      ))),
+      scalacOptions in (Compile, doc) ~= (_.filter(_.startsWith("-Xplugin"))),
```

Comment:
What / why are you filtering for here?

Reply:
it's because of this: https://github.com/DavidGregory084/sbt-tpolecat/issues/36

I later changed this to explicitly filtering out the offensive scalac options, but forgot to change here: https://github.com/raquo/Airstream/pull/68/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R24-R39